### PR TITLE
Demo prep feedback

### PR DESCRIPTION
**Pull request by tiffwong**
## Description

- Change entity icons from 21px to 20px (there was an issue with zeplin that added an extra pixel)
- Add more spacing between "Profile" and "Account" links
- Replace 'and' with '&' in UserMenu and renamed "Users" page to "Users & Grants"
- Remove missed ellipses from Managed pages
- Hide actions in Linode Landing table at an earlier breakpoint

### CODE REVIEW

File: packages/manager/src/features/linodes/LinodesLanding/LinodeActionMenu_CMR.tsx [hunk 1]
```diff
@@ -233,7 +234,7 @@ export const LinodeActionMenu: React.FC<CombinedProps> = props => {
         }
       ];
 
-      if (matchesSmDown || inTableContext) {
+      if (inTableContext || (matchesSmDown && !inTableContext)) {
```
Comment (Jskobos) [inline packages/manager/src/features/linodes/LinodesLanding/LinodeActionMenu_CMR.tsx:237]: I don't think the last `!inTableContext` is necessary, if `inTableContext` is true the logic will short circuit before it gets here.

File: packages/manager/src/features/linodes/LinodesLanding/LinodeActionMenu_CMR.tsx [hunk 2]
```diff
@@ -244,7 +245,10 @@ export const LinodeActionMenu: React.FC<CombinedProps> = props => {
         });
       }
 
-      if (matchesSmDown) {
+      if (
+        (matchesMdDown && inTableContext) ||
```
Comment (Jskobos) [inline packages/manager/src/features/linodes/LinodesLanding/LinodeActionMenu_CMR.tsx:249]: You could clean these up by assigning them to reused variables with descriptive names. Can't wait until we can delete all the inVlanContext parts, too.

**Review by Jskobos — APPROVED** [review]

**Review by johnwcallahan — APPROVED** [review]
Looks great. There's one zone where the action menu isn't visible in the Linodes table, from 600px - ~660px. I wonder if this is a case for a "custom" breakpoint to drop the Regions column (seems wasteful to drop it all the way back at 960px).

**Comment by tiffwong** [comment]
@johnwcallahan there's a ticket up for reducing the primary nav width which might take care of that breakpoint 🤔

**Pull request merged**